packages/css/src/plugins/__tests__/increase-specificity.test.ts

```diff
@@ -0,0 +1,79 @@
+import postcss from 'postcss';
+
+import { increaseSpecificity } from '../increase-specificity';
+
+const transform = (css: TemplateStringsArray) => {
+  const result = postcss([increaseSpecificity()]).process(css[0], {
+    from: undefined,
+  });
+
+  return result.css;
+};
+
+describe('increase specicifity plugin', () => {
+  it('should increase specicifity of declared classes', () => {
+    const actual = transform`
+      .foo {}
+    `;
+
+    expect(actual).toMatchInlineSnapshot(`
+      "
+            .foo:not(#\\9) {}
+          "
+    `);
+  });
+
+  it('should ignore atrules', () => {
+    const actual = transform`
+      @media {
+        .foo {}
+      }
+    `;
+
+    expect(actual).toMatchInlineSnapshot(`
+      "
+            @media {
+              .foo:not(#\\9) {}
+            }
+          "
+    `);
+  });
+
+  it('should ignore root elements', () => {
+    const actual = transform`
+      html {}
```

Comment:
What would happen if I just had this—and what's expected to happen?  Will this not increase specificity?
```tsx
css({
  button: { color: 'red' },
});
```

This is the same as the `& button` (which you could fit into the `div` test above), right?

For root/html/body or "global css" this makes sense, I guess, but for nested selectors…?  I'm less certain…

Reply:
It will. As it increases the specificity of the compiled class name.

Reply:
Yes same as div!